build.sbt

```
@@ -259,6 +260,22 @@ lazy val lmIvy = (project in file("ivy"))
     ),
   )
 
+lazy val lmCoursier = (project in file("coursier"))
+  .enablePlugins(ContrabandPlugin, JsonCodecPlugin)
+  .dependsOn(lmIvy)
+  .settings(
+    commonSettings,
+    crossScalaVersions := Seq(scala212, scala211),
+    name := "librarymanagement-coursier",
+    libraryDependencies ++= Seq(coursier, coursierCache, scalaTest, scalaCheck),
```

Only depend on `scalaTest` and `scalaCheck` for the tests?
```suggestion
    libraryDependencies ++= Seq(coursier, coursierCache, scalaTest % Test, scalaCheck % Test),
```

---

correct thanks